### PR TITLE
Enable displayRequestDuration prop for react

### LINKED ISSUE #7457: displayRequestDuration should be made available in react flavour of swagger-ui
### Content & configuration

Swagger/OpenAPI definition:
 ```yaml
 # your YAML here
 ```

 Swagger-UI configuration options:
 ```js
 SwaggerUI({
   // your config options here
displayRequestDuration : true
 })
 ```

 ```
 ?yourQueryStringConfig
 ```

### PR DESCRIPTION
**Pull request by mathis-m**
### Description

added displayRequestDuration to react props.

### Motivation and Context

Fixes #7457

### How Has This Been Tested?

not

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [x] My changes require a change to the project documentation.
- [x] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: flavors/swagger-ui-react/index.jsx [hunk 1]
```diff
@@ -102,7 +103,8 @@ SwaggerUI.propTypes = {
   defaultModelsExpandDepth: PropTypes.number,
   presets: PropTypes.arrayOf(PropTypes.func),
   deepLinking: PropTypes.bool,
-  tryItOutEnabled: PropTypes.bool
+  tryItOutEnabled: PropTypes.bool,
+  displayRequestDuration: PropTypes.bool,
 }
 
 SwaggerUI.defaultProps = {
```
Comment (char0n) [inline flavors/swagger-ui-react/index.jsx:110]: Given `displayRequestDuration` is not required prop, what about giving it a default value of `false` in defaultProps?

Comment (mathis-m) [reply flavors/swagger-ui-react/index.jsx:110]: @char0n done

File: flavors/swagger-ui-react/README.md [hunk 2]
```diff
@@ -119,6 +119,12 @@ Controls whether the "Try it out" section should start enabled. The default is f
 
 ⚠️ This prop is currently only applied once, on mount. Changes to this prop's value will not be propagated to the underlying Swagger UI instance. A future version of this module will remove this limitation, and the change will not be considered a breaking change.
 
+#### `displayRequestDuration`: PropTypes.bool
+
+Controls whether to display the duration of the request. The default is false.
```
Comment (char0n) [inline flavors/swagger-ui-react/README.md:124]:
```suggestion
Controls the display of the request duration (in milliseconds) for "Try it out" requests. The default is false.
```

Comment (char0n) [reply flavors/swagger-ui-react/README.md:124]: Just to make it consistent with https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md

**Review by char0n — APPROVED** [review]
Thanks for PR!

Given we're currently in [v4 RC release phase](https://swagger.io/blog/api-design/what%E2%80%99s-ahead-for-swaggerui-v4-and-swaggereditor-v4/) and this PR can be considered a feature, I'd postpone it and merge it right after [release/4.x](https://github.com/swagger-api/swagger-ui/tree/release/4.x) branch has been merged on master. 

Unless we can somehow argument that this PR is technically a bugfix ;]

**Comment by mathis-m** [comment]
@char0n sounds good, I have one unrelated question will there be a new petstore4.swagger.io site?

**Comment by char0n** [comment]
> @char0n sounds good, I have one unrelated question will there be a new petstore4.swagger.io site?

There is no plan currently. We'd like to merge `release/4.x` branch to master. SwaggerUI@4 stable release is about evolving our dependency tree in backward-incompatible way. 

`petstore2` and `petstore3` are designated to serve OAS 2.x and OAS 3.0.x respectively. If we have support for OAS 3.1 well probably bootstrap new `petstore31`. The number suffix after `petstore` designates version of OAS spec that SwaggerUI is build with, rather then version of SwaggerUI. Does it make sense?

**Comment by tim-lai** [comment]
@mathis-m circling back on this PR, which is now merged. Thanks for the update!
